dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05390RemoveEndpointIdForeignKeyInIntegrityResolverTablesIntegrationTest.java

```diff
@@ -0,0 +1,164 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+
+
+public class Task05390RemoveEndpointIdForeignKeyInIntegrityResolverTablesIntegrationTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190239)